validate.go

```diff
@@ -439,14 +439,16 @@ func (v *Validator) HexColor(key, value string, message ...string) {
 func (v *Validator) Len(key, value string, min, max int, message ...string) {
```

Comment:
Could you clarify the comment that it counts characters, and not byte size? There are use cases for both, so seems to me that it's good to clarify in docs. e.g.

```
// Len sets the minimum and maximum length for a string.
//
// A maximum of 0 indicates there is no upper limit. The length is counted in
// characters (runes), not bytes.
```